stock-service/src/main/java/rs/banka4/stock_service/utils/specification/SpecificationCombinator.java

```diff
@@ -0,0 +1,28 @@
+package rs.banka4.stock_service.utils.specification;
+
+import org.springframework.data.jpa.domain.Specification;
+
+public class SpecificationCombinator<T> {
```

Comment:
please don't duplicate code - move the `user_service` copy of this class into `common` and use it